### PR TITLE
FIX THE MANAGER CONFIGURATION: Part 4 rubocop fixes

### PR DESCRIPTION
**Pull request by paul-mesnilgrente**
Acceptance Criteria
===================

**Acceptance Criteria**
- bundler-audit is not returning any vulnerabilities
- `simple_form` is removed
- `mongoid` is upgraded to version 7 for consistency with other projects
- rails is upgraded to version 5.2.4
- GitHub gems forks are avoided when possible
- `therubyracer` is removed

**Notes**
- VCR and webmock are installed but not used
- capybara has an old version
- This is story was broken off from https://www.pivotaltracker.com/story/show/169468440

Background
==========

- Run a rubocop -a and it fixed everything
- No manual change

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 
-#!/usr/bin/env rake
+# !/usr/bin/env rake
+# frozen_string_literal: true
```
Comment (daveharris) [inline Rakefile]: I see this littered through our code, but not really seen it before. Do we not need it anymore?

File: .travis.yml [hunk 2]
```diff
@@ -46,18 +46,6 @@ jobs:
 
   allow_failures:
 
-    - stage: test
-      name: 'Rubocop'
-      script:
-        - bundle exec rubocop
```
Comment (hapiben) [inline .travis.yml]: Don't we want this in travis?

**Comment by hapiben** [comment]
Since you've touched rubocop, do you think you can update `rubocop` gem?

File: Rakefile (hunk 1 above)
Comment (paul-mesnilgrente) [reply Rakefile]: Do you mean the Rakefile or this change? The Rakefile is still included in newer rails project so I guess it's still needed.

File: .travis.yml (hunk 2 above)
Comment (paul-mesnilgrente) [reply .travis.yml]: The rubocop check? Yes. We are not executing any test in GitLab. It's all done in Travis. I removed the rubocop test in the allow_failures section but it's still in the normal section. Meaning that we shouldn't be able to merge if the test is failing.

See: https://github.com/DigitalNZ/supplejack_manager/blob/2dc949617e298727bcf1d197eb733fcb2a28e8a1/.travis.yml

File: Rakefile (hunk 1 above)
Comment (daveharris) [reply Rakefile]: This change. But nevermind I thought you were removing it, not adding it ...

**Comment by paul-mesnilgrente** [comment]
It's already @hapiben up to date. It was upgraded in a previous pull request :)

**Pull request merged**